code/datums/elements/turf_transparency.dm

```diff
@@ -0,0 +1,66 @@
+/datum/element/turf_transparency
+	var/show_below = FALSE
+
+///This proc sets up the signals to handle updating viscontents when turfs above/below update. Handle plane and layer here too so that they don't cover other obs/turfs in Dream Maker
+/datum/element/turf_transparency/Attach(datum/target, show_below)
+	. = ..()
+	if(!isturf(target))
+		return ELEMENT_INCOMPATIBLE
```

Comment:
It's not really intended for it but I suppose it could support it.

Reply:
I don't think its worth putting in the extra checks needed to make this work for now, but it would be possible for someone to add obj support later if they want it real bad.